docs/developer/architecture/configuration/README.md

```diff
@@ -24,7 +24,7 @@ Settings are retrieved from the ServiceExtensionContext interface.
 ```java
 class ExampleSetting {
 
-    @EdcSetting
+    @Setting
```

Comment:
Did a quick search on your branch, above this code sample in line 19 it still says @EdcSetting